### PR TITLE
Revision on English Translation

### PR DESCRIPTION
**Pull request by Caspar09**
Some Revision on English Translation

### CODE REVIEW

File: src/language/en-US.ts [hunk 1]
```diff
@@ -20,17 +21,17 @@ export const en_US = {
...
     remaining: 'Remaining',
     word: 'Word',
     publish: 'Publish',
-    anonymous_user: 'Anonymous useruser',
-    no_comment_yet: 'No comment yetcomment yet',
-    successful_operation: 'Successful operation',
-    operation_failed: 'Operatio failed',
+    anonymous_user: 'Anonymous user',
+    no_comment_yet: 'No comment yet',
+    successful_operation: 'Request Succeed',
```
Comment (decentraliser) [inline src/language/en-US.ts]: Request successful, or succeeded?

File: src/language/en-US.ts [hunk 2]
```diff
@@ -41,18 +42,18 @@ export const en_US = {
...
-    about_voting_content_description: 'About voting content description',
+    about_voting_content_description: 'Description for voting content',
     option: 'Option',
     multiple_selection: 'Multiple',
     radio: 'Radio',
     enter_the_date_for_example: 'Enter the date ,for example：',
     fee: 'Fee',
     the_default_is: 'Default',
-    the_more_you_set_the_cost_the_higher_the_processing_priority: 'The more you cost ,the higher the processing priority. (1000000gas = 1.0xem)',
+    the_more_you_set_the_cost_the_higher_the_processing_priority: 'The more you cost ,the higher the processing priority. (1000000 microxem = 1.0xem)',
```
Comment (decentraliser) [inline src/language/en-US.ts]: Increasing the fee makes the transaction processing quicker ?

File: src/language/en-US.ts [hunk 3]
```diff
@@ -142,7 +143,7 @@ export const en_US = {
     QR_code_generation_failed: 'QR code generation failed',
 
     //monitor-remote
-    Remote_title_recipt: 'Cumulative entrusted income',
+    Remote_title_recipt: 'Accumulated delegated harvest income',
```
Comment (decentraliser) [inline src/language/en-US.ts]: Harvesting

File: src/language/en-US.ts [hunk 4]
```diff
@@ -153,24 +154,24 @@ export const en_US = {
     remote_rep_list: 'Income list',
     remote_list_col1: 'Hash',
     remote_list_col2: 'Height',
-    remote_list_col3: 'Cost',
+    remote_list_col3: 'Reward',
     remote_list_col4: 'Time',
     remote_no_data: 'No data',
-    remote_repaly: 'Entrusted harvest',
+    remote_repaly: 'Delegated harvest',
```
Comment (decentraliser) [inline src/language/en-US.ts]: Harvesting

Comment (NemTechCN) [reply src/language/en-US.ts]: receive:'Receive', missed

File: src/language/en-US.ts (hunk 1 above)
Comment (Caspar09) [reply src/language/en-US.ts]: I think succeeded?

File: src/language/en-US.ts (hunk 2 above)
Comment (Caspar09) [reply src/language/en-US.ts]: Yes, this sentence need rework

**Pull request merged**